doc/classes/InputEventAction.xml

```diff
@@ -14,7 +14,7 @@
 	</tutorials>
 	<members>
 		<member name="action" type="StringName" setter="set_action" getter="get_action" default="&amp;&quot;&quot;">
-			The action's name. Actions are accessed via this [String].
+			The action's name. This is usually the name of an existing action in the [InputMap] which you want this custom event to match.
```

Comment:
"which" is a bit ambiguous here so I'd personally either remove it:
```suggestion
			The action's name. This is usually the name of an existing action in the [InputMap].
```
or:
```suggestion
			The action's name. This is usually the name of an existing action in the [InputMap], corresponding to this event.
```
... but it feels like increasing the word count to meet a quota.
I am not sure of any practical use-cases right now.

Reply:
>  I am not sure of any practical use-cases right now.

To be clear, this is the purpose of this class:

```gdscript
var fake_event = InputEventAction.new()
fake_event .name = "move_left"
Input.parse_input_event(fake_event)
```

This lets you send an event even if it wasn't actually caused by the user inputting something. You can use it for mocking, replay, or whatever game logic you have that depends on events that could originate from code (e.g. to avoid coupling to your player scene running code on it/signals directly, you send a `jump` event when you want the player to have a jumpscare in a cutscene).